core/assets/bundles/bundle_de.properties

```diff
@@ -1152,7 +1153,7 @@ rules.coreincinerates = Kern verbrennt überflüssige Materialien
 rules.disableworldprocessors = Deaktiviere Weltprozessoren
 rules.schematic = Entwürfe erlaubt
 rules.wavetimer = Wellen-Timer
-rules.wavesending = Wave Sending
+rules.wavesending = Manuelle Wellen möglich
```

Comment:
Wellen-Senden oder

Reply:
Ja, sowas in der Art hatte ich auch überlegt. Am Ende hatte ich mich doch dagegen entschieden, da es sich nicht so anhören soll, als könnte man die Wellen dann ausschließlich manuell abschicken - wie siehst du das?